doc/conf.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+from documenteer.sphinxconfig.stackconf import build_package_configs
+
+import lsst.afw
+
+_g = globals()
+_g.update(build_package_configs(
+    project_name="afw",
+    copyright="2017 Association of Univerities for "
```

Comment:
I just did a dumb substitution of `afw` -> `ap_verify`/`ap.verify` on my end and it doesn't seem to have broken documentation building...